src/ansys/hps/client/jms/schema/task_definition.py

```diff
@@ -265,3 +265,10 @@ class Meta(ObjectSchemaWithModificationInfo.Meta):
         allow_none=True,
         metadata={"description": "Enable debug logging and retain Task working directory."},
     )
+
+    working_directory = fields.String(
+        allow_none=True,
+        metadata={
+            "description": "Root working directory to be used by the evaluator for task execution"
```

Comment:
Let's stress more that it's optional

```suggestion
            "description": "Optional root working directory to be used by the evaluator for task execution."
```

same for the corresponding resource

Reply:
updated description